### DISCUSSION TITLE
Support LSA-Matting

### DISCUSSION THREAD

**Discussion by kfeng123** [open-mmlab/mmediting#1205] [General]
Hello mmedting team,

I am one of the authors of [LSA-Matting](https://github.com/kfeng123/LSA-Matting). LSA-Matting should have better performance than the supported models. And we have fully open source the project. Please feel free to add it into mmediting if you are interested.

**Comment by wangruohui** [comment]
Hello,

MMEditing is designed to be easily adaptable to different algorithms and models. Would you consider supporting your matting algorithm under our framework?

**Comment by zengyh1900** [comment]
Hi @kfeng123
All kinds of contributions are welcomed! 
We're glad to provide help if you would like to support LSA-Matting in MMEditing!
